ci-tools/test-matrix/src/html/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Caliptra test matrix</title>
```

Comment:
nit: Would be good to also have a heading at the top with this same title

Reply:
Done. See https://korran-testorg.github.io/caliptra-sw